src/debugger/common/types.js

```diff
@@ -39,6 +40,10 @@ export type PrepackLaunchArguments = {
   exitCallback: () => void,
 };
 
+export type DebuggerLaunchArguments = {
```

Comment:
This is a pretty weird name. It is not passing any launch arguments, it is passing diagnostic severity. Also, it only has one field why do we need a wrapper type here? We can directly use ```diagnosticSeverity``` in all callsites.

Reply:
The wrapper is so future features can also use this. For example, row/column indexing support, as well as sourcemaps, will all be passed through this structure. 

"Launch" is a bit misleading -- would "DebuggerConfigArguments" work better?

Reply:
Then just DebuggerConfig should be fine.